constants/userDataLevels.js

```diff
@@ -0,0 +1,21 @@
+const ACCESS_LEVEL = {
+  PUBLIC: "public",
+  INTERNAL: "internal",
+  PRIVATE: "private",
+  CONFIDENTIAL: "confidential",
+};
+
+const ROLE_LEVEL = {
+  private: ["super_user"],
+  internal: ["super_user"],
+  confidential: ["super_user"],
+};
+
+const ROLE_ACCESS = {
+  public: ["email", "phone", "chaincode"],
+  internal: ["phone", "chaincode"],
+  private: ["chaincode"],
+  confidential: [],
+};
+
```

Comment:
any specific reason for keeping the keys casing all caps in `ACCESS_LEVEL` and small case in `ROLE_LEVEL` & `ROLE_ACCESS`. we should maintain consistent across all 3 enums.